src/styles/_TreeView.scss

```diff
@@ -2,10 +2,10 @@
 
 }
   .TreeView__child {
-    border-left: 1em solid $s-color-primary9;
+    border-left: 1em solid #F2F2F2;
```

Comment:
Should this be in a var?

Reply:
I did it as a one-off because this is the only use of this particular gray, because the palette gray was slightly too dark due.